tests/openvino/test_exporters_cli.py

```diff
@@ -526,6 +526,29 @@ class OVCLIExportTestCase(unittest.TestCase):
             ]
         )
 
+    if is_transformers_version(">=", "4.49.0"):
+        TEST_4BIT_CONFIGURATIONS.extend(
+            [
+                (
+                    "image-text-to-text",
+                    "phi4mm",
+                    'int4 --group-size 8 --ratio 0.8 --sensitivity-metric "mean_activation_magnitude" '
+                    "--dataset contextual --num-samples 1 --trust-remote-code",
+                    [
+                        {"int8": 8, "int4": 42},
+                        {"int8": 1},
+                        {"int8": 8},
+                        {"int8": 2},
+                        {},
+                        {"int8": 6},
+                        {"int8": 25},
+                        {"int8": 2},
+                        {"int8": 2},
```

Comment:
would you mind adding a comment to specify which submodel does each apply to ? (to improve clarity)

Reply:
Thanks for the idea, but let me address this in a separate PR because this will require changes across a lot of test cases.